### PR TITLE
Merge Models Handler to Main

### LINKED ISSUE #7: Models handler
Implement a handler for sequelize models auto-loading.

### PR DESCRIPTION
**Pull request by Xen0Xys**

### CODE REVIEW

File: src/database/models/sample/sampleOrder.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        amount: {
+            type: DataTypes.INTEGER,
+            allowNull: false
+        },
+        author_id: {
+            type: DataTypes.INTEGER,
+            allowNull: false
+        }
+    });
+    const User = require("./sampleUser")(sequelize);
```
Comment (loicmaes) [inline src/database/models/sample/sampleOrder.js]: Don't load a model twice...
Find another way to do it

File: src/database/models/sample/sampleOrder.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const { DataTypes } = require("sequelize");
+
+module.exports = (sequelize) => {
+    const Order = sequelize.define("Order", {
```
Comment (loicmaes) [inline src/database/models/sample/sampleOrder.js]: Model name should be lowercase

File: src/database/models/sample/sampleUser.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const { DataTypes } = require("sequelize");
+
+module.exports = (sequelize) => {
+    const User = sequelize.define("User", {
```
Comment (loicmaes) [inline src/database/models/sample/sampleUser.js]: > Model name should be lowercase

Same

File: src/database/models/sample/sampleOrder.js (hunk 2 above)
Comment (Xen0Xys) [reply src/database/models/sample/sampleOrder.js]: It's sample models, they will not be here for long

File: src/database/models/sample/sampleUser.js (hunk 3 above)
Comment (Xen0Xys) [reply src/database/models/sample/sampleUser.js]: > It's sample models, they will not be here for long

Same

File: src/database/models/sample/sampleOrder.js (hunk 2 above)
Comment (loicmaes) [reply src/database/models/sample/sampleOrder.js]: Sample should help to create more... If you put a capitalized name, you will forget to put it to lowercase later...

File: src/database/models/sample/sampleUser.js (hunk 3 above)
Comment (loicmaes) [reply src/database/models/sample/sampleUser.js]: > Sample should help to create more... If you put a capitalized name, you will forget to put it to lowercase later...

Same

**Review by loicmaes — CHANGES_REQUESTED** [review]
Some changes needed...

**Review by loicmaes — APPROVED** [review]

**Pull request merged**